packages/dds/matrix/src/matrix.ts

```diff
@@ -172,12 +240,10 @@ export class SharedMatrix<T = any>
 		runtime: IFluidDataStoreRuntime,
 		public id: string,
 		attributes: IChannelAttributes,
-		_isSetCellConflictResolutionPolicyFWW?: boolean,
 	) {
 		super(id, runtime, attributes, "fluid_matrix_");
 
-		this.setCellLwwToFwwPolicySwitchOpSeqNumber =
-			_isSetCellConflictResolutionPolicyFWW === true ? 0 : -1;
+		this.setCellLwwToFwwPolicySwitchOpSeqNumber = -1;
```

Comment:
The removed constructor parameter is not a good API. It doesn't match how DDSes will be instantiated in production codepaths, which means it's quite easy to write tests which exercise scenarios that shouldn't be possible. Fortunately, none of our existing tests seem to have used that API for this purpose. There are a number of places where they're a bit creepy though and I think we should invest in fixing them at some point. E.g. we have a lot of two-client tests which set up each client by setting this bit without communicating to other clients. In the real world, whether or not FWW is set on initialization in a multi-client scenario should be communicated by way of the summary used to load the second client.